ResearchKit/Common/ORKChoiceAnswerFormatHelper.m

```diff
@@ -117,7 +121,7 @@ - (id)answerForSelectedIndexes:(NSArray *)indexes {
             [array addObject:value];
         }
     }
-    return [array copy];
+    return array.count > 0 ? [array copy] : ORKNullAnswerValue();
```

Comment:
Might want to consider returning a nil since the expected return is an array. It would be clear to other developers using the method that it will either return an array with the answers or nil.